src/deepsparse/transformers/pipelines/pipeline.py

```diff
@@ -162,7 +164,6 @@ def setup_onnx_file_path(self) -> str:
 
             self.tokenizer = AutoTokenizer.from_pretrained(
                 self.tokenizer,
-                trust_remote_code=self._trust_remote_code,
                 model_max_length=self.sequence_length,
```

Comment:
If we're removing this, we have to update the pipeline args as we're now taking in an argument that isn't being used anywhere

Reply:
@anmarques it is probably still necessary to allow for the tokenizer for some cases. i had to include this when using replit-code for example https://huggingface.co/replit/replit-code-v1-3b/blob/main/replit_lm_tokenizer.py

Reply:
Updated to code to leave trust_remote_code in place as optional argument for tokenizer